clang-tools-extra/clangd/Preamble.h

```diff
@@ -104,6 +107,8 @@ struct PreambleData {
   IncludeStructure Includes;
   // Captures #include-mapping information in #included headers.
   std::shared_ptr<const include_cleaner::PragmaIncludes> Pragmas;
+  // Information about module files for this preamble.
+  std::optional<PrerequisiteModules> DependentModulesInfo;
```

Comment:
hmm, I think the adjective "dependent" in "dependent modules" would refer to modules that depend on this, not modules that this depends on.

Maybe `UsedModules`, `RequiredModules` or just `Modules`?